docker/utils/manifest.go

```diff
@@ -10,11 +10,22 @@ import (
 
 // FIXME: Should we just use docker/distribution and docker/docker implementations directly?
 
+// GetManifestMIMETypes returns a slice of MIME types we currently support
+func GetManifestMIMETypes() []string {
+	return []string{
+		V2Schema1MIMEType,
+		V2Schema2MIMEType,
+		V2ListMIMEType,
+	}
+}
```

Comment:
Could just be a public `var`? OTOH that’s perhaps too risky.

Drop the `Get` prefix though :)